src/plugins/vis_types/gauge/public/convert_to_lens/configurations/metric.ts

```diff
@@ -14,15 +14,14 @@ import {
 } from '@kbn/visualizations-plugin/common';
 import { GaugeVisParams } from '../../types';
 
-export const getConfiguration = (
+export const getMetricConfiguration = (
```

Comment:
Since gauge/goal are using a "predefined" palette, what if we migrate to a predefined palette too? Like:
- Green to Red -> Status
- Yellow to Red -> Warm
- Grays -> Gray
- Green -> Positive
- Red -> Negative
- Blues -> Cool

In this way we can have nice colors that don't affect the readability of the chart as the existing one are doing

Reply:
Good idea! I had to change a bit of code around but I think this is a good refactor if you want to take a look at 8daf98b975d1135a5bf5a5ba50b8d4c9a3fad28d.

The final palette is labelled as custom because there are custom steps but I think this is the best way to do it.

Reply:
I see, I'm fine with that. Go with it